main_logic/omni_realtime_client.py

```
@@ -1104,81 +1104,96 @@ async def stream_image(self, image_b64: str) -> None:
-                    "role": "user",
-                    "content": [
-                        {
-                            "type": "input_text",
-                            "text": instructions
-                        }
-                    ]
-                }
+        if skipped:
+            self._skip_until_next_response = True
+
+        # 通过 conversation.item.create 添加用户消息，再触发响应
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)